components/Login/index.js

```diff
@@ -0,0 +1,115 @@
+import { signIn, signOut, useSession } from "next-auth/react";
+import styled from "styled-components";
+import Image from "next/image";
+import { useState } from "react";
+
+export default function Login({ isHomepage }) {
+  const { data: session } = useSession();
+  const [isHovered, setIsHovered] = useState(false);
```

Comment:
There are some improvements I can propose to your code : 

3. Add a loading state: This improves the user experience by showing feedback when an action is being processed.

 const [loading, setLoading] = useState(false);